Doc/whatsnew/3.12.rst

```diff
@@ -74,6 +74,14 @@ Important deprecations, removals or restrictions:
 New Features
 ============
 
+* Add :ref:`perf_profiling` through the new command-line option
+  :option:`-X perf <-X>`, as well as the new
```

Comment:
What about the envvar?  Also in previous examples there was no space between `-X` and `perf` (not that it matters much, but we might as well be consistent).

Reply:
yes, we should mention the environment variable here.